httpbin/core.py

```diff
@@ -649,6 +648,91 @@ def links(n):
     return redirect(url_for('link_page', n=n, offset=0))
 
 
+@app.route('/file/<path:p>', methods=('GET', 'POST',))
```

Comment:
allows also for easier testing of this to know each method only does one thing, either a post or get.  less prone to mistakes.